x/supplier/client/cli/helpers_test.go

```diff
@@ -32,3 +54,197 @@ func networkWithSupplierObjects(t *testing.T, n int) (*network.Network, []shared
 	cfg.GenesisState[types.ModuleName] = buf
 	return network.New(t, cfg), supplierGenesisState.SupplierList
 }
+
+// TODO_CONSIDERATION: perhaps this (and/or other similar helpers) can be refactored
+// into something more generic and moved into a shared testutil package.
+// TODO_TECHDEBT: refactor; this function has more than a single responsibility,
+// which should be to configure and start the test network. The genesis state,
+// accounts, and claims set up logic can probably be factored out and/or reduced.
+func networkWithClaimObjects(
+	t *testing.T,
+	sessionCount int,
+	supplierCount int,
+	appCount int,
+	// TODO_THIS_COMMIT: hook up to genesis state generation...
```

Comment:
TODO_IN_THIS_COMMIT. 

Still intending todo in this commit?

Reply:
I'm gonna remove the comment and the unused `serviceCount` param. My original thinking was that it would make sense for this function to parameterize services as well, but the need hasn't quite arisen yet so I would put it off at this point.